yarn.lock

```diff
@@ -2,205 +2,332 @@
 # yarn lockfile v1
 
```

Comment:
This can be removed from git and gitignored

Reply:
> @sudhasane This PR need refinement. It should not have been branched from such an old branch really and there are many quality issues. I'd like to see a link to a passing build using the nightly Jenkinsfile but before that, the functionality of the nightly pipeline is not as desired. This pipeline file will only all tests against one environment despite the parameters for the test urls. Please address these issues and ensure that the pipeline can target any environment and defaults to aat.

we run nightly tests only on AAT thats the requirement from hmcts